### DISCUSSION TITLE
Is it possible to force an Update to 4.9, even in condition "IncompatibleOperatorsInstalled"?

### DISCUSSION THREAD

**Discussion by C3PO-github** [okd-project/okd#1087] [Q&A] [answered]
Hi everyone!

We have some operators in our 4.8.21 cluster that are incompatible to Kubernetes 1.22 (for example strimzi v0.21). 

We have given all teams a deadline by which they have to make sure the operators are updated. In the likely case that some users of the platform were not able to update the operators used, how can we force an update of the platform?

Despite manual acknowledgment, no 4.9.x version can be selected as target version in the administration interface if an incompatible operator is installed.

Steps to reproduce:
Step 1: Install strimzi v0.21
Step 2: Set the "acknowledgment" mentioned by the guide https://docs.okd.io/4.9/updating/updating-cluster-prepare.html
Step 3: Change the update channel to 4.9 
Step 4: Try to update. You can not select any 4.9.x (greyed out)
Step 5: delete the "blocking" oiperator
Step 6: Now you can perform the update

**Comment by vrutkovs** [comment, marked as answer]
>Is it possible to force an Update to 4.9, even in condition "IncompatibleOperatorsInstalled"

No, this will render operator ususable, so you might as well uninstall the operator

**Reply by C3PO-github** [reply]
Unfortunately, that was exactly our fear.  Deleting the operators on our side is probably out of the question, so we have to think of a solution to force the teams to update their operators beforehand.

Thanks for the information anyway!
